README.md

```diff
@@ -141,144 +144,149 @@ function countStore (state, emitter) {
-on a team, no matter the size, to fully understand how an application is laid
-out. And once an application is built, we want it to be small, performant and
-easy to reason about. All of which makes for easy to debug code, better results
-and super smiley faces.
+Choo doesn't believe that bigger is better. Big APIs, large complexities, long files see like omens of impending userland complexity. Choo is made so everyone on a team, no matter the size, can fully understand how an application is laid out. And once an application is built, Choo is designed to remain small, performant and easy to reason about. These decisions make it easy to debug code, provide better results
+and put smiles on faces.
 
 ## Events
-At the core of Choo is an event emitter, which is used for both application
-logic but also to interface with the framework itself. The package we use for
-this is [nanobus](https://github.com/choojs/nanobus).
+
+Choo is an event emitter, which is used for both application
+logic and also to interface with the framework itself. [Nanobus](https://github.com/choojs/nanobus) is used to provide this functionality.
 
 You can access the emitter through `app.use(state, emitter, app)`, `app.route(route,
 view(state, emit))` or `app.emitter`. Routes only have access to the
 `emitter.emit` method to encourage people to separate business logic from
 render logic.
 
-The purpose of the emitter is two-fold: it allows wiring up application code
-together, and splitting it off nicely - but it also allows communicating with
-the Choo framework itself. All events can be read as constants from
-`state.events`. Choo ships with the following events built in:
+### The purpose of the emitter is two-fold
 
-### `'DOMContentLoaded'`|`state.events.DOMCONTENTLOADED`
-Choo emits this when the DOM is ready. Similar to the DOM's
-`'DOMContentLoaded'` event, except it will be emitted even if the listener is
-added _after_ the DOM became ready. Uses
+-  It allows wiring up application code together and splitting it off nicely.
+-  It allows communicating with the __Choo__ framework itself. All events can be read as constants from `state.events`. 
+
+### Choo ships with the following events built in
+
+#### `'DOMContentLoaded'`|`state.events.DOMCONTENTLOADED`
+
+Emitted when the DOM is ready. Similar to the DOM's
+`'DOMContentLoaded'` event, except it will be emitted even if the listener is added _after_ the DOM became ready. Uses
 [document-ready](https://github.com/bendrucker/document-ready) under the hood.
 
-### `'render'`|`state.events.RENDER`
-This event should be emitted to re-render the DOM. A common pattern is to
+#### `'render'`|`state.events.RENDER`
+
+Emitted to re-render the DOM. A common pattern is to
 update the `state` object, and then emit the `'render'` event straight after.
-Note that `'render'` will only have an effect once the `DOMContentLoaded` event
+Note that `'render'` will only have an affect once the `DOMContentLoaded` event
```

Comment:
The original phrasing was grammatically correct, whereas your edit looks wrong to me.

Reply:
Maybe what you meant was: "Note that `render` will only affect the `DOMContentLoaded` event"